frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -748,6 +749,7 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
                     });
                   }}
                   rules={[
+                    layerNameRules[0] as { min: number }, // Ensuring minimum length
                     {
                       validator: (newReadableLayerName) =>
                         validateReadableLayerName(
```

Comment:
I think the additional rules should be implemented within `validateReadableLayerName` (or maybe `checkLayerNameForInvalidCharacters` for the `.`).

Reply:
sure thanks for the suggestion. Should be implemented now :)